README.md

```diff
@@ -82,6 +82,30 @@ Syncing buffer_overflow
 Success!
 ```
 
+## 5. Verify & pull challenges
+
+If, at any point, changes may have been made to a challenge through the CTFd UI by an admin. To verify that your challenge.yml file matches remote contents, you can use:
+
+```
+❯ ctf challenge verify [challenge.yml | DIRECTORY] [--verify-files] [--verify-defaults]
+```
+
+If the `--verify-files` flag is set, challenge files will be downloaded and the binary files will be compared.
+
+If the `--verify-defaults` flag is set, challenge files will be verified to make sure they include default optional keys present on CTFd.
+If you want to pull down the latest version of the challenge, and its challenge files, you can use:
```

Comment:
See below regarding just showing basic concepts of the idea

Reply:
👍